### PR TITLE
[Merged by Bors] - feat: the complement of a null set has full measure

### PR DESCRIPTION
**Pull request by YaelDillies**
From PFR

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Seems plausible to me, thanks!
maintainer merge

File: Mathlib/MeasureTheory/Measure/NullMeasurable.lean [hunk 1]
```diff
@@ -280,6 +280,12 @@ theorem measure_union₀' (hs : NullMeasurableSet s μ) (hd : AEDisjoint μ s t)
 theorem measure_add_measure_compl₀ {s : Set α} (hs : NullMeasurableSet s μ) :
     μ s + μ sᶜ = μ univ := by rw [← measure_union₀' hs aedisjoint_compl_right, union_compl_self]
 
+lemma measure_of_measure_compl_eq_zero₀ (hs : NullMeasurableSet s μ) (h : μ sᶜ = 0) :
```
Comment (sgouezel) [inline Mathlib/MeasureTheory/Measure/NullMeasurable.lean:283]: You shouldn't need `hs` here: `s^c` is contained in a measurable set of measure zero, so `s` contains a measurable set of full measure.

Comment (sgouezel) [reply Mathlib/MeasureTheory/Measure/NullMeasurable.lean:283]: And in fact you already know that `s` is null-measurable, as `s^c` is by `h`...

Comment (YaelDillies) [reply Mathlib/MeasureTheory/Measure/NullMeasurable.lean:283]: Whoops yeah 🙈 

I was trying to find something along those lines, but I think I got lost when trying a separate generalisation (which didn't work out) simultaneously

**Comment by sgouezel** [comment]
bors r+
Thanks!

**Pull request closed without merging**